completions/awscli.completion.sh

```diff
@@ -1 +1 @@
-[[ -x "$(which aws_completer)" ]] &>/dev/null && complete -C "$(which aws_completer)" aws
+_omb_util_command_exists aws_completer && complete -C "$(_omb_util_command_exists aws_completer)" aws
```

Comment:
The latter call of which is used to get the path but not just test the existence of the command.